cms/templates/settings.html

```diff
@@ -292,6 +292,28 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                 <span class="tip">${_("Information for prospective students")}</span>
               </header>
               <ol class="list-input">
+
+                % if about_page_editable:
+                <li class="field" id="field-course-title">
+                  <label for="course-title">${_("Course Title")}</label>
+                  <input type="text" id="course-title">
```

Comment:
We need to prepopulate this with the course title that is entered when creating the course.